lib/phoenix/router.ex

```diff
@@ -64,6 +64,12 @@ defmodule Phoenix.Router do
       MyApp.Router.Helpers.page_url(conn_or_endpoint, :show, "hello", some: "query")
       "http://example.com/pages/hello?some=query"
 
+  If the route contains glob-like patterns, parameters for those have to be given as
+  list and will not be escaped:
+
+      MyApp.Router.Helpers.dynamic_path(conn_or_endpoint, :show, ["/some/route"])
```

Comment:
Oh, those should be escaped. If they are not escaped, it is a bug. The whole point of receiving a list is so we escape them. :)